lumberjack.go

```diff
@@ -165,7 +166,15 @@ func (l *Logger) Write(p []byte) (n int, err error) {
 func (l *Logger) Close() error {
 	l.mu.Lock()
 	defer l.mu.Unlock()
-	return l.close()
+
+	err := l.close()
+
+	if l.millCh != nil {
+		close(l.millCh)
+		l.wg.Wait()
```

Comment:
after waiting, you need to set millCh to nil, so if someone calls close again, it won't panic, trying to close the same channel.

Reply:
yes